src/NetMQ/packages.NetMQ.config

```diff
@@ -1,5 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="AsyncIO" version="0.1.18.0" targetFramework="net40" />
-  <package id="JetBrains.Annotations" version="10.0.0" targetFramework="net40" />
+  <package id="JetBrains.Annotations" version="10.0.0" targetFramework="net40" developmentDependency="true"/>
```

Comment:
See above. Because I removed the explicit dependencies from nuspec, nuget will now correctly bring in the real dependencies ignoring all developer dependencies.

I will add the same for .Net 3.5